Minor fixes and update of angular library versions

smeyn opened the pull request:
## Pull Request Template
Please, go through these steps before you submit a PR.

1. Make sure that your PR is not a duplicate.
2. **Before** submitting the PR, review your changes:
    * Adjust the code to the existing [style](https://google.github.io/styleguide).
    * Add a set of unit tests for the changes and ensure they all pass.
    * Add accompanying README.md file with instructions on usage. See [awesome-readme](https://github.com/matiassingers/awesome-readme) for good examples of high-quality READMEs.
    * Add a link to your contribution in the top-level [README](https://github.com/GoogleCloudPlatform/professional-services/blob/master/README.md) (alpha-order).
    * Add [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) license headers with an up-to-date copyright date attributed to `Google LLC`.
    * Remove unnecessary LICENSE files. There's no need to include an additional license since all repository submissions are covered by the top-level Apache 2.0 [license](https://github.com/GoogleCloudPlatform/professional-services/blob/master/LICENSE).

3. **After** these steps, you're ready to open a pull request.
    * Give a descriptive title to your PR.
    * Provide a description of your changes.
    * Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
<br/>

**PLEASE REMOVE THIS TEMPLATE BEFORE SUBMITTING**

dparrish commented on tools/bq-visualizer/src/app/big-query.service.ts:
```diff
@@ -24,14 +24,19 @@ export class BigQueryService {
 
     const token = this.oauthService.getAccessToken();
-    const url = bqUrl(`/${projectId}/jobs/${realid}`, {access_token: token});
+    console.log(token);
```
I think this debugging should be removed

smeyn replied:
At this stage I still find situations where this is useful. So I am leaving itin until the code matures further.

dparrish commented on tools/bq-visualizer/src/app/bq_query_plan.ts:
```diff
@@ -14,13 +14,27 @@ export class BqQueryPlan {
+      return;
     }
+    this.isValid = true;
+    console.log(this);
```
remove debugging?

smeyn replied:
done

dparrish commented on tools/bq-visualizer/src/app/bq_query_plan.ts:
```diff
@@ -169,16 +194,59 @@ export class BqQueryPlan {
   /** Return a formatted text of all details minus the steps. */
   getStageStats(node: QueryStage): string {
-    const result = {};
+    /*const result = {};
```
Remove instead of commenting

dparrish replied:
This is still here as a commented-out block. Does it need to be here?

dparrish commented on tools/bq-visualizer/src/app/job/job.component.ts:
```diff
@@ -91,12 +91,19 @@ export class JobComponent implements OnDestroy {
             detail => {
-              console.log('Got raw plan', detail);
-              this.planSelected.emit(new BqQueryPlan(detail, this.logSvc));
+              // console.log('Got raw plan', detail);
```
remove

dparrish commented on tools/bq-visualizer/src/app/rest_interfaces.ts:
```diff
@@ -65,16 +65,22 @@ interface Query {
   timeline: Timeline;
 }
-
+interface ReservationUsage {
```
Add a blank line between top-level definitions

dparrish commented on tools/bq-visualizer/src/app/timing-display/timing-display.component.ts:
```diff
@@ -29,14 +29,21 @@ export class TimingDisplayComponent implements OnInit {
-    if (!this.plan) return;
+    if (!this.plan) {
+      const gantt = document.getElementById('Gantt');
+      if ((gantt) && (gantt.childNodes.length > 0)) {
```
Too many parentheses.

`if (gantt && gantt.childNodes.length > 0) {`

smeyn replied:
done

dparrish commented on tools/bq-visualizer/src/tsconfig.app.json:
```diff
@@ -3,11 +3,14 @@
+      "node"
+    ],
+    "strictPropertyInitialization": false,
+    "strictNullChecks": false,
```
JSON doesn't allow a trailing comma on the last element of a map.

dparrish commented on tools/bq-visualizer/src/tsconfig.spec.json:
```diff
@@ -7,7 +7,8 @@
     ],
-    "strictPropertyInitialization": true
+    "strictPropertyInitialization": false,
+    "strictNullChecks": false,
```
JSON doesn't allow a trailing comma on the last element of a map.

smeyn commented:
Addressed changes requested by David. 
In the mean time added minor bug fixes and added individual Aache 2 licence headers as laid out in pull request template.
